test/TestHelper.js

```diff
@@ -215,6 +215,27 @@ export async function setEditorValue(editor, value) {
   await act(() => {});
 }
 
+
+export async function expectEventually(fn) {
```

Comment:
Proposal: Rename this utility to `eventually`.

Makes for cleaner tests, I think:

```javascript
await eventually(() => {
  expect(mockComponent).to.have.been.calledWith(
  sinon.match({
    variables: [
      {
        name: 'OutputVariable_0svilsd',
        info: 'Written in Task_2'
      }
    ]
  });
});
```

Reply:
Isn't this what https://testing-library.com/docs/dom-testing-library/api-async#waitfor does already? Comes with `@testing-library/preact`.

Reply:
I did not know about this one, I'll check it out

Reply:
Works exactly like it, I changed it to use the testing utility instead 👍